openshift/sso74-x509.build.yaml

```diff
@@ -52,17 +52,18 @@ objects:
   spec:
     lookupPolicy:
       local: false
-- apiVersion: v1
-  kind: ImageStream
-  metadata:
-    annotations: null
-    creationTimestamp: null
-    labels:
-      shared: "true"
-    name: ${NAME}-base
-  spec:
-    lookupPolicy:
-      local: false
+# SH: remove base chain build, use configmap
+# - apiVersion: v1
```

Comment:
why comment out

Reply:
we are not doing docker chain build now, that piece of configuration is now moved to a configmap (as mentioned in this PR) Just keeping those as comment for now, in case we need to roll back.